apps/web/screens/Grants/SearchResults/SearchResults.tsx

```diff
@@ -0,0 +1,470 @@
+                              onClick: () => {
+                                router.push(
+                                  linkResolver(
+                                    'styrkjatorggrant',
+                                    [grant?.applicationId ?? ''],
+                                    locale,
+                                  ).href,
+                                )
+                              },
+                              icon: 'arrowForward',
+                            }}
+                            detailLines={[
+                              grant.dateFrom && grant.dateTo
+                                ? {
+                                    icon: 'calendar' as const,
+                                    text: `${format(
+                                      new Date(grant.dateFrom),
+                                      'dd.MM.',
+                                    )}-${format(
+                                      new Date(grant.dateTo),
+                                      'dd.MM.yyyy',
+                                    )}`,
+                                  }
+                                : null,
+                              {
+                                icon: 'time' as const,
+                                //todo: fix when the text is ready
+                                text: 'Frestur til 16.08.2024, kl. 15.00',
+                              },
+                              grant.categoryTag?.title
+                                ? {
+                                    icon: 'informationCircle' as const,
+                                    text: grant.categoryTag.title,
+                                  }
+                                : undefined,
+                            ].filter(isDefined)}
+                          />
+                        )}
+                      </Box>
+                    )
+                  })}
+                </Inline>
+              ) : undefined}
+              {!grants?.length && !error && !loading && (
+                <Box
+                  display="flex"
+                  alignItems="center"
+                  justifyContent="center"
+                  background="white"
+                  borderWidth="standard"
+                  borderRadius="xl"
+                  borderColor="blue200"
+                  flexDirection={['columnReverse', 'columnReverse', 'row']}
+                  columnGap={[2, 4, 8, 8, 20]}
+                  paddingY={[5, 8]}
+                  paddingX={[3, 3, 5, 10]}
+                  rowGap={[7, 7, 0]}
+                >
+                  <Box display="flex" flexDirection="column" rowGap={1}>
+                    <Text variant={'h3'} as={'h3'} color="dark400">
+                      {formatMessage(m.search.noResultsFound)}
+                    </Text>
+                    <Text whiteSpace="preLine">
+                      Borem ipsum dolor sit amet, consectetur adipiscing elit.
```

Comment:
I know this is a work in progress but can we at least have this something that is ok that users see?